index.js

```diff
@@ -149,6 +163,8 @@ Fluenture.prototype.pipe = function pipe(f) {
 //.
 //. Fluent [`alt`](https://github.com/fluture-js/Fluture#alt).
 Fluenture.prototype.alt = unaryDispatcher (fluent, alt);
+Fluenture.prototype['fantasy-land/alt'] =
+  fantasyLandUnaryDispatcher (fluent, 'fantasy-land/alt');
```

Comment:
With the first argument removed, this might fit on one line. ;)

Reply:
If not, there's also an opportunity to bake the `fantasy-land/`-prefix into the `fantasyLandUnaryDispatcher` function.

Reply:
Have a look, even with your recommendation the lines were too long.  I just created a helper function that allowed me to shorten the strings.  Hopefully it's an acceptable alternative if you really want them to fit on one line :-D.

Reply:
Haha, it's not that important. But it can stay now that it's there! :)